docs/tutorial/part-zero/index.md

```diff
@@ -10,48 +10,72 @@ Before you start building your first Gatsby site, you’ll need to familiarize y
+- On **MacOS** with homebrew installed, run `brew install node`. If you don't want to install it through homebrew, download the latest Node.js version from [the official Node.js website](https://nodejs.org/en/), double click on the downloaded file and go through the installation process.
 
-Once you have followed the installation steps, make sure everything was installed properly:
+- On **Windows**, download and install the latest Node.js version from [the official Node.js website](https://nodejs.org/en/)
 
-### Check your Node.js installation
+- On **Linux**:
 
-1.  Open up your terminal.
-2.  Run `node --version`. (If you’re new to the command line, “run `command`” means “type `node --version` in the command prompt, and hit the Enter key”. From here on, this is what we mean by “run `command`”).
-3.  Run `npm --version`.
+  Install curl and nvm (Node Version Manager). nvm is used to manage node and all its associated versions.
+
+  - Ubuntu, Debian, and other `apt` based distros:
+    - Run `sudo apt-get install curl` to install curl which allows you to transfer data and download additional dependencies.
+    - After it finishes installing, run `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash` to download the latest nvm version.
+    - To confirm this has worked, use the following command. `nvm --version`. The output should be a version number.
+  - Arch, Manjaro and other `pacman` based distros:
+    - These distros come installed with curl, so you can use that to download nvm.
+      `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash`
+    - Before using nvm, you need to install additional dependencies by running `sudo pacman -S grep awk tar git`.
+    - To confirm this has worked, use the following command. `nvm --version`. The output should be a version number.
+  - Fedora, RedHat, and other `dnf` based distros:
+    - These distros come installed with curl, so you can use that to download nvm.
+      `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash`
+    - To confirm this has worked, use the following command. `nvm --version`. The output should be a version number.
+
+If the Linux distribution you are using is not listed here, please find instructions on the web.  
+ _💡 If when installing a package, it asks for confirmation, type `y` and press enter._
 
-The output of each of those commands should be a version number. Your versions may not be the same as those shown below! If entering those commands doesn’t show you a version number, go back and make sure you have installed Node.js.
+#### Set default Node.js version
+
+When nvm is installed, it does not default to a particular node version. You’ll need to install the version you want and give nvm instructions to use it. This example uses the latest release of version 10, but more recent version numbers can be used instead.
+
+```shell
+nvm install 10
+nvm use 10
+```
```

Comment:
```suggestion
```

To confirm that this worked, you can run `npm --version` and `node --version`. The output should look similar to the screenshot below, showing version numbers in response to the commands. 
```

Reply:
Still need to add this.